zipline/utils/calendars/calendar_helpers.py

```diff
@@ -73,7 +73,7 @@ def next_scheduled_day(date, last_trading_day, is_scheduled_day_hook):
         dt += delta
         if is_scheduled_day_hook(dt):
             return dt
-    return None
+    raise NoFurtherDataError(msg='Cannot find next day after %s' % date)
```

Comment:
Are there any consumers besides the scheduled events that need updating to handle this error?